src/ITensorNetworks.jl

```diff
@@ -82,6 +82,7 @@ include("specialitensornetworks.jl")
 include("renameitensornetwork.jl")
 include("boundarymps.jl")
 include("beliefpropagation.jl")
+include("ContractionTreeToGraph.jl")
```

Comment:
I think we should name this file `contraction_tree_to_graph.jl`. File names are generally lower case unless the file defines a module.